src/graph/test/GroupByLimitTest.cpp

```diff
@@ -0,0 +1,114 @@
+/* Copyright (c) 2019 vesoft inc. All rights reserved.
+ *
+ * This source code is licensed under Apache 2.0 License,
+ * attached with Common Clause Condition 1.0, found in the LICENSES directory.
+ */
+
+#include "base/Base.h"
+#include "graph/test/TestEnv.h"
+#include "graph/test/TestBase.h"
+#include "graph/test/TraverseTestBase.h"
+#include "meta/test/TestUtils.h"
+
+DECLARE_int32(load_data_interval_secs);
+
+namespace nebula {
+namespace graph {
+
+class GroupByLimitTest : public TraverseTestBase {
+protected:
+    void SetUp() override {
+        TraverseTestBase::SetUp();
+        // ...
+    }
+
+    void TearDown() override {
+        // ...
+        TraverseTestBase::TearDown();
+    }
+};
+
+TEST_F(GroupByLimitTest, SyntaxError) {
+    {
+        cpp2::ExecutionResponse resp;
+        auto query = "LIMIT 5";
+        auto code = client_->execute(query, resp);
+        ASSERT_EQ(cpp2::ErrorCode::SUCCEEDED, code);
+    }
+}
+
+TEST_F(GroupByLimitTest, LimitTest) {
+    {
+        cpp2::ExecutionResponse resp;
+        auto &player = players_["Marco Belinelli"];
+        auto *fmt = "GO FROM %ld OVER serve YIELD $$.team.name as name"
+                    " | ORDER BY $-.name | LIMIT 5";
+        auto query = folly::stringPrintf(fmt, player.vid());
+        auto code = client_->execute(query, resp);
+        ASSERT_EQ(cpp2::ErrorCode::SUCCEEDED, code);
+        std::vector<std::tuple<std::string>> expected = {
+                {"76ers"},
+                {"Bulls"},
+                {"Hawks"},
+                {"Hornets"},
+                {"Kings"},
+        };
+        ASSERT_TRUE(verifyResult(resp, expected, false));
```

Comment:
let's test a case : the return result number is larger than limit count.

eg. return 5 results but limit 6,1.